test_communication/test/action_server_py.py

```diff
@@ -180,6 +182,8 @@ def execute_goal(goal_handle):
     except BaseException:
         print('Exception in action server:', file=sys.stderr)
         raise
+    except ExternalShutdownException:
+        sys.exit(1)
```

Comment:
@ivanpauno this one has the same ordering issue!

Reply:
argh, fixed in https://github.com/ros2/system_tests/pull/490/commits/9f0bb2a7ff36c54d67c50b53627f5edc9456a5a3.